static/js/main.js

```diff
@@ -0,0 +1,19 @@
+const adviceBtn = document.querySelector('.generate-advice-btn');
+const adviceIdSpan = document.querySelector('.advice-id');
+const adviceQuote = document.querySelector('.advice-quote');
+
+const generateAdvice = () => {
+    fetch("https://api.adviceslip.com/advice", { cache: 'no-cache' })
+        .then((response) => response.json())
+        .then((data) => {
```

Comment:
Add a check to see if object actually exists and contains the data that you are looking for.